docs/quick-start.md

```diff
@@ -1,7 +1,7 @@
 # Quick-start
 
-The following describes the Minimal steps to get started with CRI Resource
-Manager.
+The following describes the minimum number of steps to get started with CRI
+Resource mManager.
```

Comment:
typo: 'mManager'

Reply:
Thanks for catching that typo! Latest commit fixed it.